LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@Pilchie Could you verify that this is the correct License file?

Copied it from https://github.com/aspnet/AspNetCore